client/app/src/main/java/com/example/mrides/Activity/CreateRouteActivity.java

```diff
@@ -9,6 +9,7 @@
 import android.app.Dialog;
 import android.app.ProgressDialog;
 import android.content.Context;
+import android.content.SyncStatusObserver;
 import android.content.pm.PackageManager;
 import android.graphics.Bitmap;
```

Comment:
Unused import statement on line 14 and 16 for `import android.graphics.drawable.BitmapDrawable; ` and `import android.graphics.Bitmap;`